src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (a) => {
```

Comment:
The getSum function should take two arguments directly, not through another function.

Reply:
How is it possible? In the tests we use getSum(x)(y), it is equal to:
const firstF = getSum(x)
const secondF = firstF(y)

How I can take 2 arguments directly?
Thank you!